### PR TITLE
MaxMind: Fix - PAPP-23253 SemGrep Pipeline should all pass & Run pre-commit

### PR DESCRIPTION
**Pull request by paulnawat**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [X] Testing of all the changes has been performed (for bug fixes / features)
- [X] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [X] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [X] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [X] Verify all checks are passing.
- [X] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [X] Do NOT submit updates to dependencies unless it fixes an issue.
- [X] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [ ] New App
- [X] Bugfix
- [ ] Feature
- [X] Code style update (formatting, renaming)
- [X] Refactoring (no functional changes, no api changes)
- [X] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Fix SemGrep pipeline failure.
- Run pre-commit on all files to ensure the app sticks to our convention.

## What is the current behavior? (OPTIONAL)
- SemGrep pipeline is failing.
- Some code styles are out of sync.

## What is the new behavior? (OPTIONAL)
- SemGrep pipeline is passing.
- Pre-commit does not raise any issue.

---
Thanks for contributing!

### CODE REVIEW

File: maxmind.json [hunk 1]
```diff
@@ -408,4 +408,4 @@
             "versions": "EQ(*)"
         }
     ]
-}
\ No newline at end of file
+}
```
Comment (paulnawat) [inline maxmind.json:411]: This is added as part of `pre-commit run --all-files`

File: maxmind_connector.py [hunk 2]
```diff
@@ -226,7 +226,7 @@ def _should_download_new_db(self):
         self.debug_print('Checking if the current database is up to date.')
         db_url = DB_DOWNLOAD_URL.format(self._license_key)
 
-        r = requests.head(db_url)
+        r = requests.head(db_url, timeout=DEFAULT_REQUEST_TIMEOUT)
```
Comment (paulnawat) [inline maxmind_connector.py:229]: Timeout is added to fix the SemGrep piepline.

File: readme.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
 [comment]: # "Auto-generated SOAR connector documentation"
 # MaxMind
 
-Publisher: Splunk
```
Comment (paulnawat) [inline readme.md:4]: The change to this `readme.md` is auto generated by `pre-commit run --all-files`.

File: maxmind_consts.py [hunk 4]
```diff
@@ -47,3 +47,5 @@
 MMDB_FILE = "GeoLite2-City.mmdb"
 MMDB_TAR_FILE = "GeoLite2-City.tar.gz"
 DB_DOWNLOAD_URL = "https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-City&license_key={0}&suffix=tar.gz"
+
+DEFAULT_REQUEST_TIMEOUT = 30  # in seconds
```
Comment (paulnawat) [inline maxmind_consts.py:51]: I'm using the same timeout value as this app https://github.com/splunk-soar-connectors/forescoutcounteract/blame/2d63a3fd8c39c4e7d069e34022c08e5e85928820/forescoutcounteract_consts.py#L83

File: maxmind_connector.py (hunk 2 above)
Comment (paulnawat) [reply maxmind_connector.py:229]: Fix is recommended by this log https://github.com/splunk-soar-connectors/maxmind/runs/4612649991?check_suite_focus=true

**Review by brian-phantom — APPROVED** [review]

**Pull request merged**